.github/CODEOWNERS

```
@@ -1,21 +1,21 @@
-*  @snyk/hammerhead @snyk/snyk-open-source
+*  @snyk/cli @snyk/snyk-open-source
```

```suggestion
*  @snyk/cli
```

I think by default there should be one code owner, and it should be the CLI team. I'd consider also removing everything below this line.

This doesn't mean we would stop accepting contributions from these other teams. It just means that we expect all contributions to go through a PR review and get a +1 from CLI team.

I think such reviews are necessary to prevent breaking changes and other unpleasant surprises for customers.